src/common/databaseFilters.ts

```diff
@@ -0,0 +1,31 @@
+import { ServerSideFilterMethod } from "@/components/Tables/Filters";
+import { nullPostcodeDisplay } from "./format";
+import { DbClientRow, DbParcelRow } from "@/databaseUtils";
+
+export const fullNameMethodHelper = <DbData extends DbClientRow | DbParcelRow>(
```

Comment:
It would be really good if you could check that this conforms to [VFB-211](https://softwiretech.atlassian.net/jira/software/c/projects/VFB/boards/862?selectedIssue=VFB-211)

i.e., if we set columnLabel to be a string that **is not** a DB column for the specific table, what happens?